Sources/Core/Presenter/Channels/ChannelsPresenter+Rx.swift

```diff
@@ -43,6 +43,18 @@ public extension Reactive where Base == ChannelsPresenter {
             .do(onNext: { [weak base] _ in base?.rx.removeFromItems(channelPresenter) })
             .asDriver(onErrorJustReturn: .empty)
     }
+    
+    /// Delete a channel and remove a channel presenter from items.
+    ///
+    /// - Parameters:
+    ///   - channelPresenter: a channel presenter.
+    ///   - clearHistory: checks if needs to remove a message history of the channel.
+    func delete(_ channelPresenter: ChannelPresenter) -> Driver<Channel> {
```

Comment:
You forgot to add `clearHistory` parameter.

Reply:
I removed the comment. Delete endpoint doesn't have clearHistory parameter.